package-build.el

```diff
@@ -798,8 +827,10 @@ in `package-build-archive-dir'."
          (desc (let ((default-directory source-dir))
                  (package-build--desc-from-library
                   name version commit files))))
-    (unless (string-equal (downcase (concat name ".el"))
-                          (downcase (file-name-nondirectory file)))
+    ;; TODO For ttl-mode.
```

Comment:
What's the resolution for the TODO? Or is this a general precaution that is specifically needed by at least that package?

Reply:
I just accidentally left that comment intact.

> Or is this a general precaution that is specifically needed by at least that package?

Yes, that.